WMF Framework/CommonStrings.swift

```diff
@@ -168,4 +168,5 @@ public class CommonStrings: NSObject {
     
     public static let genericErrorDescription = WMFLocalizedString("fetcher-error-generic", value: "Something went wrong. Please try again later.", comment: "Error shown to the user for generic errors with no clear recovery steps for the user.")
     
+    public static let findAndReplaceHeaderTitle = WMFLocalizedString("find-replace-header", value: "Find and replace", comment: "Find and replace header title.")
```

Comment:
would it make sense to keep this string in `SectionEditorViewController`? looks like it's only used in that file

Reply:
yep you're right, done!